test_scripts/API/VehicleData/Refactor_Fuel_Information/009_App_version_is_equal_to_fuelRange_parameter_version_is_listed_in_DB_and_API.lua

```diff
@@ -0,0 +1,39 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Referring to this comment https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2413#discussion_r451132043, I think this new test should be removed and the existing test `VehicleDataVersioning/002_VD_parameter_in_API_DB_app_version_equal_parameter_version.lua` should be expanded to include the fuelRange param